clusters/app.ci/assets/logging.yaml

```diff
@@ -79,6 +79,23 @@ data:
       region = "us-east-1"
       stream_name = "{{ kubernetes.pod_node_name }}"
       encoding.codec = "json"
+
+    # Release-controller logs
+    [transforms.release_controllers]
+      type = "filter"
+      inputs = ["kubernetes"]
+      condition.type = "check_fields"
+      condition."kubernetes.pod_namespace.eq" = "ci"
+      condition."kubernetes.pod_name.starts_with" = "release-controller-"
+      condition."kubernetes.container_name.eq" = ["controller"]
+
+    [sinks.rc_aws_cloudwatch_logs]
+      type = "aws_cloudwatch_logs"
+      inputs = ["release_controllers"]
+      group_name = "app-ci-logs/{{ kubernetes.pod_labels.app }}"
```

Comment:
This will create a new log group and it will keep creating a new group when the label will change. Let's just use the same group `app-ci-pod-logs` or if you need a different retention policy let's create a new one with a stable name.

Reply:
There are currently 8 distinct release-controller variants.  The pod label is what is distinguishing them.  I grouped them this way to ensure that all relevant logs remain together and not get all jumbled together into on massive bucket for all.

Reply:
Yeah and I do not want them in our group because they are unstructured :P